js/notifications.js

```diff
@@ -92,7 +99,7 @@
                 window.nodeNotifier.notify({
                     title: title,
                     message: message,
-                    sound: false
+                    sound: shouldPlayNotificationSound,
```

Comment:
This is interesting. We only use `nodeNotifier` on Windows 7. So I presume we haven't tested this. What's the theory, at least?

Reply:
Good call, I reverted it since I couldn’t test it! The docs say this is macOS only and we never polyfill notifications there. I’ll get back to this whenever I get a chance to set up a full Windows 7 VM. Or maybe someone from the community can play around with this and let us know what the Windows 7 behavior is 😄